src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -232,7 +232,14 @@ to this directory.",
               .long("chunk-bytes")
               .required(false)
               .default_value(&format!("{}", 3 * 1024 * 1024))
-        )
+        ).arg(
+          Arg::with_name("thread-count")
+              .help("Number of threads to use for uploads and downloads")
+              .takes_value(true)
+              .long("thread-count")
+              .required(false)
+              .default_value("1")
```

Comment:
Rather than defaulting to 1, consider defaulting to `None` and propagating that optional value down to the threadpool. Both the `CpuPool` and `Runtime` support using "number of CPU cores" as their default if a thread count isn't specified.

Reply:
Alas `grpcio::Environment::new` does not :(